docs/jwst/outlier_detection/outlier_detection_spec.rst

```diff
@@ -1,50 +1,24 @@
 .. _outlier-detection-spec:
 
-Outlier Detection for Slit-like Spectroscopic Data
-==================================================
+Slit-like Spectroscopic Data
+============================
 
 This module serves as the interface for applying ``outlier_detection`` to slit-like
-spectroscopic observations.  The code implements the
-basic outlier detection algorithm used with HST data, as adapted to JWST
-spectroscopic observations.
+spectroscopic observations. The algorithm is very similar to the
+:ref:`imaging algorithm <outlier-detection-imaging>`, and much of the same code is used.
+Please refer to those docs for more information.
+A :ref:`Stage 3 association <asn-level3-techspecs>`,
+which is loaded into a :py:class:`~jwst.datamodels.ModelContainer` object,
+serves as the input and output to this step, and the :py:class:`~jwst.datamodels.ModelContainer`
+is converted into a :py:class:`~jwst.datamodels.ModelLibrary` object where processing is
+shared with the imaging mode.
```

Comment:
This comment is a little unclear - is the container created in the outlier detection step, or passed from the pipeline?  Also the "processing is shared" comment makes it sound like there is some kind of computational sharing, but I think you meant the code is shared.  Maybe say something like "converted into a ModelLibrary for compatibility and consistency with imaging inputs"?